tests/test_sf_arcgis_app.py

```diff
@@ -0,0 +1,82 @@
+# pylint: disable=redefined-outer-name
+"""Tests for examples/app_sf_arcgis.py"""
+import json
+import pytest
+import jsend
+from falcon import testing
+import examples.app_sf_arcgis
+
+@pytest.fixture()
+def client():
+    """ client fixture """
+    return testing.TestClient(examples.app_sf_arcgis.run())
+
+def test_get_fields_by_address_example(client):
+    """Test get_fields_by_address_example"""
+    address = '1650 mission street'
+
+    response = client.simulate_get(
```

Comment:
this call seems like it would make an actual request to arcgis.
shouldn't sf_arcgis_sdk.sf_arcgis.requests.get be mocked?

Reply:
I'm doubling use for this test to also do e2e testing. Tests that use mocks are in `test_sf_arcgis.py`